weblate/trans/machine/weblatetm.py

```diff
@@ -56,11 +94,11 @@ def download_translations(self, source, language, text, unit, user):
         """Download list of possible translations from a service."""
         matching_units = Unit.objects.same_source(unit)
 
-        return [
-            self.format_unit_match(munit, 100)
+        return list(set([
+            self.format_unit_match(munit, text)
```

Comment:
It can stay 100 here - this service returns only exact matches.

Reply:
that's not true (at least with Weblate 2.18 as shown in WeblateOrg/weblate#1791)